test.sh

```diff
@@ -1,7 +1,8 @@
-#! /bin/bash
+#!/usr/bin/env bash
 
 cd "$(dirname "$0")"
 bin="./target/debug/huniq"
+test -e "$bin" || bin=$(which huniq)
```

Comment:
Oh, these are the integration tests so they should always refer to the local build. Otherwise we might end up running our integration tests on the system huniq.